public/tenants/pampers/config.ts

```diff
@@ -17,10 +17,7 @@ const config = {
     locale: 'en_US',
   },
   home: {
-    title: 'Pampers ❤️ Wald',
     descriptionTitle:"Bäume pflanzen für eine nachhaltigere Zukunft",
```

Comment:
This is left despite existing as a resource now as well. Is the resource used or this?

Reply:
I need to find a solution to add html tags in json file, if we do that we will not have to add different keys for these things.

Reply:
Not sure if this is helpful, but have you though about using https://react.i18next.com/latest/trans-component?